html/captive-portal/templates/pki.html

```diff
@@ -0,0 +1,73 @@
+[% title = i18n("PacketFence Certificate Initialisation System") %]
+[% INCLUDE header.html
+    body_attributes = 'onload="$(\'firstfield\').focus();"';
+%]
+
+        <div id="about" class="underlined">
+          <img src="/content/images/registration.png" alt="Registration" />
+          <h1>[% i18n("Certificate generation for EAP TLS connection") %]</h1>
+          <p>[% i18n("guest network disclaimer") %]</p>
+          <hr/>
+        </div>
+
+        [%# Error %]
+        [% IF txt_validation_error.defined %]<div class="error"><p>[% txt_validation_error %]</p></div>[% END %]
+        <form method="post" action="[% post_uri %]">
+          <input type="hidden" name="destination_url" value="[% destination_url %]" />
+
+          <div class="input">
+            <span>[% i18n("Username") %]</span>
+            <input class="field" name="certificate_cn" id="firstfield" type="text" value="[% certificate_cn | html %]" />
+            
+            <span>[% i18n("Password") %]</span>
+            <input class="field" name="certificate_pwd" type="text" value="[% certificate_pwd | html %]" />
+
+            <span>[% i18n("Email") %]</span>
+            <input class="field" name="certificate_email" type="text" value="[% certificate_email | html %]" />
+
+          </div>
+
+          <!--<div class="input">
+              <span>[% i18n("Service") %]</span>
+              <select name="service">
+                <option value="">[% i18n("Select your service") %]</option>
+                    [% FOREACH profile IN profile_list %]<option value="[% profile %]"[% IF service == profile %] selected[% END %]>[% profile %]</option>
+                    [% END %]
+              </select>
+          </div>-->
+
+          [% skipFields = {username => undef, password => undef, profile => undef, email => undef} %]
+          [% FOREACH field IN mandatory_fields %]
```

Comment:
This is the only place in your PR that you use mandatory_fields. Are you using this piece of code ?